_posts/2023-02-07-Announcing-Data-Prepper-2.1.0.md

```diff
@@ -0,0 +1,251 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.1.0"
+authors:
+- TODO - Krishna
+- nsifmoh
+- dlv
+date: 2023-02-07 15:00:00 -0500
+categories:
+  - releases
+---
```

Comment:
Please add front matter. 

**Meta Keywords**: Data Prepper 2.1.0, Open Telemetry metrics, detect anomalies in the logs and traces, sampling and rate-limiting
**Meta description**: Data Prepper 2.1.0 improves performance and stability with enhancements such as generated metrics, anomaly detection, sampling and limiting, and more.